home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -0,0 +1,438 @@
+openapi: 3.0.0
+info:
+  title: Voting
+  description: This element contains voting units created by owner based on the OpenAPI 3.0 specification.
+  version: 1.0.0
+tags:
+  - name: voting
+    description: Votings management operations
+    externalDocs:
+      description: Find out more
+      url: http://swagger.io
+paths:
+  /api/votings :
+    get:
+      tags:
+        - Voting
+      summary: Returns a list of active votings.
+      responses:
+        '200':    # status code
+          description: Successfully returns a list of active votings.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/VotingDTO'
+
+        '400':
+          description: Invalid request
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+  /api/variantsByVoting/{id}:
+    get:
+      tags:
+        - Voting
+      summary: Returns a voteslist in chosen voting .
+      parameters:
+        - name: id
+          in: path
+          required: true
+          description: voting id
+          schema:
+            type : integer
+            format: int64
+            example: 1
+      responses:
+        '200':    # status code
+          description: Successfully returns a list of votes.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/VotingVariantsDTO'
+
+
+        '400':
+          description: Invalid request
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+  /api/voting/{id}:
```

Comment:
What is the difference between `/votings` and `/voting/{id}`? Seems to me it should be `/votings/{id}`